src/Core/Migration/V6_7/Migration1752499887UpdateAppRequestedPrivileges.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1752499887UpdateAppRequestedPrivileges extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1752499887;
+    }
+
+    public function update(Connection $connection): void
+    {
+        if ($this->columnExists($connection, 'app', 'requested_privileges')) {
```

Comment:
as `Migration1740563553AddAppRequestedPrivileges` is not part of any release yet, we can simply adjust that migration

Reply:
but this will not be the version deployed, as you need this fix in it, right? so it should not matter :thinking: correct me, if I am wrong :grin:

Reply:
when you want that all staging shops are different than production ones 🙈